File: Codigos/Ejercicio2.java

import java.util.ArrayList;
import java.util.Scanner;
 
public class Ejercicio2 {
    public static void main(String[] args) {
        ArrayList<Double> notas = new ArrayList<>();
        Scanner sc = new Scanner(System.in);
        double suma = 0;
 
        System.out.println("=== REGISTRO DE 6 NOTAS ===");
        for (int i = 0; i < 6; i++) {
            System.out.print("Ingrese nota " + (i+1) + " (0-10): ");
            double nota = sc.nextDouble();
            notas.add(nota); // Agregar al ArrayList
            suma += nota;
        }
 
        double promedio = suma / 6;
        System.out.println("\n--- Notas Registradas ---");
        for (int i = 0; i < notas.size(); i++) {
            System.out.println("Nota " + (i+1) + ": " + notas.get(i));
        }
        System.out.printf("Promedio General: %.2f%n", promedio);
        sc.close();
    }
}

File: Codigos/ejercicio2.cpp
#include <iostream>
#include <vector>
using namespace std;
 
int main() {
    vector<float> notas;
    float nota;
    float suma = 0;
 
    cout << "=== REGISTRO DE 6 NOTAS ===" << endl;
    for (int i = 0; i < 6; i++) {
        cout << "Ingrese nota " << i+1 << " (0-10): ";
        cin >> nota;
        notas.push_back(nota); // Agregar nota al vector
        suma += nota;
    }
 
    // Calcular el promedio
    float promedio = suma / 6;
 
    cout << "\n--- Notas Registradas ---" << endl;
    for (int i = 0; i < notas.size(); i++) {
        cout << "Nota " << i+1 << ": " << notas[i] << endl;
    }
    cout << "Promedio General: " << promedio << endl;
    return 0;
}
